app/api-requests/third-party-api/base.js

```diff
@@ -0,0 +1,19 @@
+const wreck = require('@hapi/wreck')
+const session = require('../../session')
+const { tokens } = require('../../session/keys')

```

Comment:
could we maybe call this "rpa-api/base.js` as i guess the authorization header party might not be generic enough?

Reply:
Maybe best to just rename the folder "third-party-api" to "rpa-api" as all files within it are kind of RPA specific.